docs/development/oot.md

```diff
@@ -5,21 +5,49 @@ page_id: oot
 
 It is possible to have an out-of-tree build of parts of the crazyflie firmware. This enables developers to work on elements without worrrying about merging it with the full code base. 
 
-# App layer.
-Technically the app layer is an example of an out of tree build. Follow the [app layer instructions](/docs/userguides/app_layer.md) for this.
-
-# OOT estimators
+# General out-of-tree build process
 In a seperate folder make a Makefile which contain the following content:
 
+```Makefile
+CRAZYFLIE_BASE := [LOCATION OF THE CRAZYFLIE FIRMWARE]
+
+#
+# We override the default OOT_CONFIG here, we could also name our config
+# to oot-config and that would be the default.
+#
+OOT_CONFIG := $(PWD)/config
+
+include $(CRAZYFLIE_BASE)/tools/make/oot.mk
 ```
-CFLAGS += -DOOT_ESTIMATOR
 
-VPATH += src/
-PROJ_OBJ += estimator_out_of_tree.o
+This will make the crazyflie-firmware build system look for a `Kbuild` file in your folder.
```

Comment:
Currently the kbuild is unresponsive if you don't make it inside of an example folder or oot estimator folder... Is it possible to have some kind of warning or feedback that this is what is happening?

Reply:
> Currently the kbuild is unresponsive if you don't make it inside of an example folder or oot estimator folder... Is it possible to have some kind of warning or feedback that this is what is happening?

Could you give me an example? I do not quite understand.

Reply:
Strange, I can select it fine.

Reply:
From where are you doing `make menuconfig` ?

Reply:
the main crazyflie-firmware repo